io.ossia.score.metainfo.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>io.ossia.score</id>
+  <metadata_license>CC-BY-SA-4.0</metadata_license>
+  <project_license>GPL-3.0</project_license>
+  <name>ossia score</name>
+  <summary>Sequencer for media artists</summary>
+  <developer id="io.github.ossia">
+    <name>ossia.io</name>
+  </developer>
+  <description>
+    ossia score is a sequencer for audio-visual artists, designed to enable the creation of interactive shows, museum installations, intermedia digital artworks, interactive music and more in an intuitive user interface.
+    
+    <ul>
+      <li>Sequence OSC, MIDI, DMX, sound, video and more, between multiple software and hardware. </li>
+      <li>Script and live-code with JavaScript, ISF Shaders, Faust, PureData or C++.</li>
+      <li>Leverage IoT protocols such as CoAP or MQTT for industrial-grade automation.</li>
+      <li>Interact with joysticks, Wiimotes, Leapmotions, Web APIs and BLE sensors.</li>
+      <li>Integrate programs from a wealth of creative programming languages such as Structure Synth, Context-Free Art and Bytebeat.</li>
+      <li>Load any kind of audio or video format and process visuals through Spout, Syphon, NDI or Shmdata.</li>
+    </ul>
+  </description>
+  <launchable type="desktop-id">io.ossia.score.desktop</launchable>
+  <content_rating type="oars-1.1" />
+  <url type="bugtracker">https://github.com/ossia/score/issues</url>
+  <url type="contact">https://discord.gg/8Hzm4UduaS</url>
+  <url type="contribute">https://ossia.io/project.html</url>
+  <url type="donation">https://opencollective.com/ossia</url>
+  <url type="help">https://ossia.io/docs.html</url>
+  <url type="homepage">https://ossia.io</url>
+  <url type="vcs-browser">https://github.com/ossia/score</url>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://github.com/ossia/score/raw/master/docs/score.png?raw=true</image>
+      <caption>Sequence all kinds of media in ossia score</caption>
+    </screenshot>
+  </screenshots>
+  <releases>
+    <release version="3.2.2" date="2024-07-01">
+    <url type="details">https://github.com/ossia/score/releases/tag/v3.2.2</url>
+      <description>
+        ossia score 3.2.2 is a bugfix release with mostly non-Linux-related improvements.
```

Comment:
this is invalid markup. Text must be at least in `<p/>` tag.

Reply:
hmm, when I did this appstream-validate gave me an issue : 
```
$ appstreamcli validate --explain io.ossia.score.metainfo.xml
I: io.ossia.score:11: description-first-word-not-capitalized
   The description line does not start with a capitalized word, project name or number.
```

can I safely ignore it ? (the software name is explicitly lower-case and should never be spelled Ossia Score or Ossia score for instance)